### PR TITLE
docs: add html_theme_options

### PR DESCRIPTION
**Pull request by bendichter**
Add edit button to docs and a link to the GitHub repo

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -66,6 +66,18 @@
 }
 add_module_names = False
 
+html_theme_options = {
+    "use_edit_page_button": True,
+    "icon_links": [
+        {
+            "name": "GitHub",
+            "url": "https://github.com/catalystneuro/neuroconv",
```
Comment (CodyCBakerPhD) [inline docs/conf.py:74]:
```suggestion
            "url": "https://github.com/NeurodataWithoutBorders/nwb-guide",
```

**Comment by bendichter** [comment]
image

**Comment by CodyCBakerPhD** [comment]
The 'Edit on GitHub' button leads to a 404 from https://nwb-guide--651.org.readthedocs.build/en/651/

but works fine on https://neuroconv.readthedocs.io/en/main/

Maybe it needs the `html_context` as well?

**Comment by bendichter** [comment]
I was thinking that was a preview issue but you might be right

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**